### DISCUSSION TITLE
Invalid extended configuration does not lead to error

### DISCUSSION THREAD

**Discussion by DOliana** [renovatebot/renovate#12593] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest docker image (29.3.5)

### Please select which platform you are using if self-hosting.

Azure DevOps Server

### Describe the bug

We use a default.json and dedicated configs in each of our repos.
We now observed that when the default-config is invalid (in our case a typo), there is no error in the logs when running renovate even when loglevel is set to debug. It seems to just use its default config.

The expected behaviour would be the same as when running with an invalid config. We would expect something like
```
INFO: Found renovate config errors (repository=Extended/default-config)
 "errors": [
         {
           "topic": "Configuration Error",
           "message": "Invalid configuration option: packageRules[1].descsription"
         },
         {
           "topic": "Configuration Error",
           "message": "Invalid configuration option: packageRules[2].descsription"
         }
       ]
```

### Relevant debug logs

No relevant logs...

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment]
`default.json` is not a Renovate configuration file name, only a suggested file name for presets.

It would not be a good idea if every time Renovate ran on a repo it looked for files called `default.json` and created an error in the repo if it wasn't a valid Renovate file.

Instead, you can use the bundled `renovate-config-validator` command as part of your CI checks in repos containing presets. You can see in this issue an example of running it against all .json files in a repo: https://github.com/renovatebot/renovate/issues/12272#issue-1033638154

**Reply by timmkrause** [reply]
@rarkins But it seems that it's an implemented convention, also the [docs](https://docs.renovatebot.com/config-presets/) mention `default.json` all the time. Should I create a PR to update the docs to guide people to "make it right" in the first place?

**Reply by rarkins** [reply]
No, I think you're misunderstanding my reply above. There's nothing wrong with using `default.json` for presets. It would be wrong for Renovate to automatically warning people if a `default.json` had invalid config.
